lib/gtfs/components/ShowAllRoutesOnMapFilter.js

```diff
@@ -34,11 +33,13 @@ export default class ShowAllRoutesOnMapFilter extends Component<Props> {
     // performance might suffer (server should be fine). This is a somewhat
     // arbitrary limit. NL has over 4M shape records.
     const tooManyShapeRecords = version.feedLoadResult.shapes.rowCount > 1000000
+    const noShapes = version.feedLoadResult.shapes.rowCount === 0
     return (
       <Checkbox
         checked={showAllRoutesOnMap}
-        disabled={fetchStatus.fetching}
+        disabled={noShapes || fetchStatus.fetching}
         onChange={this.onChange}
+        title={noShapes && this.messages('noShapes')}
```

Comment:
On some browsers this gets rendered as "undefined". A ternary with a blank string as else case might work better

Reply:
updated in https://github.com/ibi-group/datatools-ui/pull/929/commits/a76c93b03a12bbd04917e418a7a7dd6a74524327